### PR TITLE
adapted lhe submission to pwg and fixed bug in seed conversion

### PR DESCRIPTION
**Pull request by selvaggi**

### CODE REVIEW

**Comment by clementhelsens** [comment]
I would prefer to have a specific case for Powheg rather than making it the default. Also before merging the bug fix for the seed, we need to make sure we will not produce more of the old samples.

**Comment by selvaggi** [comment]
> I would prefer to have a specific case for Powheg rather than making it the default. Also before merging the bug fix for the seed, we need to make sure we will not produce more of the old samples.

Replaced the option gp by gp_mg and gp_pw to account for different job id and seed. 
As far as checking whether seed will not produce old files all over again, I think it is inevitable. 
Most likely with the old method duplicates have already been generated. We simply have to generate new samples now...

File: bin/run.py [hunk 1]
```diff
@@ -51,8 +51,13 @@
 
     sendjobGroup = parser.add_argument_group('type of jobs to send')
     sendjobGroup.add_argument('--type', type=str, required = '--send' in sys.argv and '--reco'  in sys.argv , help='type of jobs to send', choices = ['lhep8','p8'])
-    sendjobGroup.add_argument('--typelhe', type=str, required = '--send' in sys.argv and '--LHE'  in sys.argv , help='type of jobs to send', choices = ['gp','mg'])
+    sendjobGroup.add_argument('--typelhe', type=str, required = '--send' in sys.argv and '--LHE'  in sys.argv , help='type of jobs to send', choices = ['gp_mg','gp_pw','mg'])
     sendjobGroup.add_argument('-q', '--queue', type=str, default='workday', help='lxbatch queue (default: workday for HTCONDOR)', choices=['1nh','8nh','1nd','2nd','1nw','espresso','microcentury','longlunch','workday','tomorrow','testmatch','nextweek'])
+    sendjobGroup.add_argument('--priority', type=str, default='group_u_FCC.local_gen', help='condor queue priority (default: group_u_FCC.local_gen)')
+    sendjobGroup.add_argument('--ncpus', type=str, default='1', help='number of CPUs (1CPU=2Gb of RAM)')
```
Comment (clementhelsens) [inline bin/run.py]: can you also add those two new options to readme?

File: bin/send_lhep8.py [hunk 2]
```diff
@@ -106,13 +111,29 @@ def send(self, force):
             if self.decay not in self.para.decaylist[pr_noht]:
                 print 'decay ==%s== does not exist for process ==%s=='%(self.decay,self.process)
                 sys.exit(3)
+        '''
+        ##FIXME have to make py card a mandatory argument
```
Comment (clementhelsens) [inline bin/send_lhep8.py]: this is done or?

File: bin/send_lhep8.py [hunk 3]
```diff
@@ -143,15 +164,16 @@ def send(self, force):
...
 
             if nbjobsSub == self.njobs: break
-
+            
             tmpf=None
             with open(All_files[i], 'r') as stream:
                 try:
                     #tmpf = yaml.load(stream, Loader=yaml.FullLoader)
-                    tmpf = yaml.load(stream)
+                    tmpf = yaml.load(stream, Loader=yaml.FullLoader)
```
Comment (clementhelsens) [inline bin/send_lhep8.py]: that depends on the python version, but should like this for newest ones, is this also changed to all the other places where yaml are loaded?

File: config/param_FCC.py [hunk 4]
```diff
@@ -54,15 +54,10 @@
...
-##init script for FCCSW
-#stack=fccsw_dir+'init_fcc_stack.sh'
-stack=fccsw_dir+'setup.sh'
-##FCCSW dir
-#fccsw=fccsw_dir+'fccsw/0.8.2/x86_64-slc6-gcc62-opt/'
-fccsw='/afs/cern.ch/work/v/vavolkl/public/fcc.cern.ch/sw/fccsw/v0.10/x86_64-centos7-gcc62-opt/'
-fccsw='/cvmfs/fcc.cern.ch/sw/releases/fccsw/0.10/x86_64-centos7-gcc62-opt/'
+#stack='/cvmfs/fcc.cern.ch/sw/latest/setup.sh'
+fccsw = '/afs/cern.ch/work/s/selvaggi/private/FCCSW_dev/'
+stack='/afs/cern.ch/work/s/selvaggi/private/FCCSW_dev/init.sh'
```
Comment (clementhelsens) [inline config/param_FCC.py]: can't stay like this for prod

File: bin/run.py (hunk 1 above)
Comment (selvaggi) [reply bin/run.py]: done.

File: bin/send_lhep8.py (hunk 2 above)
Comment (selvaggi) [reply bin/send_lhep8.py]: yes, not needed since default pythia_card specified

Comment (selvaggi) [reply bin/send_lhep8.py]: added instruction in README

File: config/param_FCC.py (hunk 4 above)
Comment (selvaggi) [reply config/param_FCC.py]: fixed

**Comment by clementhelsens** [comment]
@selvaggi , can we merge this PR or do you expect to push other updates?

File: config/param_FCC.py [hunk 5]
```diff
@@ -17,8 +17,10 @@
 yamldir='/afs/cern.ch/work/h/helsens/public/FCCDicts/yaml/FCC/'
 
 #heppy and proc lists
-heppyList = '/afs/cern.ch/work/h/helsens/public/FCCDicts/FCC_heppySampleList_VERSION.py'
-procList = '/afs/cern.ch/work/h/helsens/public/FCCDicts/FCC_procDict_VERSION.json'
+heppyList = '/afs/cern.ch/work/s/selvaggi/private/EventProducer/FCC_heppySampleList_VERSION.py'
```
Comment (clementhelsens) [inline config/param_FCC.py]: points to your private list in a private area

File: config/param_FCC.py [hunk 6]
```diff
@@ -17,8 +17,10 @@
...
+#heppyList = '/afs/cern.ch/work/h/helsens/public/FCCDicts/FCC_heppySampleList_VERSION.py'
+#procList = '/afs/cern.ch/work/h/helsens/public/FCCDicts/FCC_procDict_VERSION.json'
+procList = '/afs/cern.ch/work/s/selvaggi/private/EventProducer/FCC_procDict_VERSION.json'
```
Comment (clementhelsens) [inline config/param_FCC.py]: same here

**Pull request merged**